packages/channels/botpress-channel-messenger/src/index.js

```diff
@@ -70,6 +70,7 @@ module.exports = {
     verifyToken: { type: 'string', required: false, default: '' },
     enabled: { type: 'bool', required: true, default: true },
     hostname: { type: 'string', required: false, default: '', env: 'MESSENGER_HOST' },
+    freezeProfile: { type: 'bool', required: false, default: false },
```

Comment:
@dmk23 , it'd be great to have this key explained in the README for the messenger package.
Also not sure which item on the least is it related to?

Reply:
@epaminond I can add an explanation. What specific file should it go to?

I grouped this key at the top of the config, between authentication and profile details, because it governs whether all the following fields would be uploaded to Facebook